src/test/java/PlayerCopierTest.java

```diff
@@ -0,0 +1,161 @@
+import org.junit.jupiter.api.Assertions;
+import org.junit.jupiter.api.Test;
+
+class PlayerCopierTest {
+    @Test
+    public void PlayerDetailsCopied() {
+        Player p = new Player("Player 1", 'W');
+        Player p2 = p.copy();
+
+        Assertions.assertSame("Player 1", p2.getName());
+        Assertions.assertSame('W', p2.getColor());
+    }
+
+    @Test
+    public void PlayerMemoryDifferent() {
+        Player p = new Player("Player 1", 'W');
+        Player p2 = p.copy();
+
+        Assertions.assertNotSame(p, p2);
+    }
+
+    @Test
+    public void PlayerPointValuePreserved() {
+        Player p = new Player("Player 1", 'W');
+
+        Piece pawn1 = new Piece();
```

Comment:
Consider refactoring with a private method to reduce this repetition. It probably isn't a bad idea to use a `PieceFactory`, as that is the only way pieces are ever created anyway.